Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/CheckList.java

```diff
@@ -39,4 +46,31 @@ public CheckList(By optionsNamesLocator) {
     public CheckList(By optionsNamesLocator, By allOptionsNamesLocator) {
         super(optionsNamesLocator, allOptionsNamesLocator);
     }
+
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JCheckList.class, ICheckList.class))
+            return;
```

Comment:
Please surround with curly braces. And try to avoid if-else (and similar code structures) statements without using curly.